File: warehouse-api/tools/generate_password.go
package main

import (
	"fmt"
	"log"

	"golang.org/x/crypto/bcrypt"
)

func main() {
	password := "password123"

	// Generate hash
	hash, err := bcrypt.GenerateFromPassword([]byte(password), bcrypt.DefaultCost)
	if err != nil {
		log.Fatal(err)
	}

	fmt.Println("Password:", password)
	fmt.Println("Bcrypt Hash:", string(hash))
	fmt.Println("\nUpdate your seed data with this hash!")
}

File: warehouse-api/tools/generate_password.js
// Generate bcrypt hash for passwords
// Usage: node generate_password.js

const bcrypt = require('bcryptjs');

const password = 'password123';

// Generate hash
const hash = bcrypt.hashSync(password, 10);

console.log('\n=================================');
console.log('Password:', password);
console.log('Bcrypt Hash:', hash);
console.log('=================================');
console.log('\nUpdate your seed data (002_seed_data.sql) with this hash!');
